src/partisan_hyparview_peer_service_manager.erl

```diff
@@ -57,14 +57,19 @@
          code_change/3]).
 
 %% temporary exceptions
--export([delete_state_from_disk/0]).
+-export([delete_state_from_disk/0,
+         delete_epoch_from_disk/0]).
```

Comment:
I assume that it is specifically for the sets of active & passive views, but if it is okay to handle them together I will change it.